OpenUpSA/wazimap-ng-ui
# displaying choropleth from the rich data panel

## Description
Displaying choropleth of a subindicator on a chart on the rich data panel

## Related Issue
https://github.com/OpenUpSA/wazimap-ng-ui/issues/281

## How to test it locally
* expand rich data panel
* click on a `Display Choropleth` button on any row of any chart table
* confirm that the rich data panel is closed and choropleth is displayed

## Screenshots
a sample button that displays choropleth when clicked

![image](https://user-images.githubusercontent.com/53019884/117369631-d7fad200-aecd-11eb-9b36-a88cf77eea36.png)


## Changelog

### Added

### Updated
* `profile.js`, `profile_loader.js`, `category.js`, `subcategory.js` and `indicator.js` to add bubbleevent and create `parents` object
* `chart.js` to append a temporary button to the chart tables and manage its click event
* `setup/choropleth.js` to display choropleth when the button in a chart table is clicked
* `setup/miscelements.js` to close rich data panel when the button in a chart table is clicked

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
